c_plus_plus/src/model/stream/utils/threads/thread.cpp

```diff
@@ -57,6 +57,7 @@ void Thread::threadExecution()
         std::cout << "Unknown exception during thread execution" << std::endl;
     }
 
+    isAbortRequested_ = false;
     isRunning_ = false;
 }

```

Comment:
Why we did not use a QThread instead of making our own? Maybe there is a good reason tho